backend/src/election/structs.rs

```diff
@@ -27,6 +28,13 @@ pub enum ElectionCategory {
     Municipal,
 }
 
+/// Election status (limited for now)
+#[derive(Serialize, Deserialize, ToSchema, Clone, Debug, PartialEq, Eq, Hash, Type)]
+pub enum ElectionStatus {
+    DataEntryInProgress,
+    DataEntryFinished,
```

Comment:
We don't seem to have functionality to get an election from `DataEntryInProgress` to `DataEntryFinished`.

I can click "Invoerfase afronden" and then download the *proces-verbaal*, but that doesn't change the status of the election.

Reply:
Yes we have not implemented this flow yet (see Figma). I set one of the elections in the fixtures and mock to finished so you can test how it looks. I can set the election to finished upon downloading the report, but this is not the moment this should happen so I was hesitant to do this.

Reply:
Good point. Implementing the status change on a button that's only temporarily on that page would not really be better.

Arguably we should have kept the statuses limited to the single `DataEntryInProgress` for now, but let's not revert all the work you've done.